README.md

```diff
@@ -1,2 +1,36 @@
-# rfcs
-RFCs for Strapi future changes
+# Strapi RFCs
+
+Some important changes in Strapi require some thoughts to be put into the design phase before starting workin on a PR.
```

Comment:
Recent important changes in Strapi require thought be put into the design phase of your PR, even prior to starting to work on it.